tsconfig.base.json

```diff
@@ -9,6 +9,7 @@
     "noEmitOnError": false,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
+    "resolveJsonModule": true,
```

Comment:
I don't think we need to change this in the base config, most plugins don't use JSON imports

Reply:
doesn't hurt anything, but it is left over from a previous attempt at removing tslib for the test.